easybuild/easyconfigs/g/GDB/GDB-7.9-missing-makeinfo.patch

```diff
@@ -0,0 +1,13 @@
+# Don't let the make fail when makeinfo is not found
+diff -ur gdb-7.9.orig/missing gdb-7.9/missing
+--- gdb-7.9.orig/missing	2015-02-19 12:58:08.000000000 +0100
++++ gdb-7.9/missing	2015-03-11 14:22:52.000000000 +0100
+@@ -204,7 +204,7 @@
+ 
+ # Propagate the correct exit status (expected to be 127 for a program
+ # not found, 63 for a program that failed due to version mismatch).
+-exit $st
++exit 0
```

Comment:
hmm, this looks kind of a generic change?

you're basically telling the GDB build procedure here that `makeinfo` *is* there? or possible that all tools it checks for are there?